types/src/front/assistant/templates.ts

```diff
@@ -177,3 +180,46 @@ export const generateTailwindBackgroundColors = (): string[] => {
   });
   return tailwindColors;
 };
+
+export function getAgentActionConfigurationType(
```

Comment:
This was just moved from `front/lib/api/assistant/templates.ts` but no changes on it!